CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.15.2+16
+
+- Updated SDK version to 2.0.0-dev.17.0
```

Comment:
Nit: Other entries below use `*` instead of `-` for bullets.

Reply:
Ah, good catch. I'll have to check all the PRs for that.
(We apparently don't have a single consistent CHANGELOG format).